client/src/views/Project.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <html>
+    <!-- this needs some serious styling -->
+    <div class="main">
+      <div class="row" style="padding-top: 20px">
+        <div class="col text-info1">
+          <h1>{{ this.project.name }}</h1>
+          <p v-if="this.project.looking_for_new_members">
+            is looking for applicants!
+          </p>
+          <div class="row">
+            <p
+              v-for="(f, i) in JSON.parse(this.project.search_filters)"
```

Comment:
Lets try not to do this in the html, but either in the mounted or a separate function. Some error handling for values that cannot go through a JSON parse should also be put into place.

Reply:
i see